tests/test_Canvas.py

```diff
@@ -0,0 +1,55 @@
+"""This module contains unit tests for the Catalog class"""
+
+__author__ = "Siavash Yasini"
+__email__ = "yasini@usc.edu"
+
+import pytest
+
+
+def test_clean_canvas(test_canvas):
```

Comment:
Minor thing — a nice pattern is to name your tests things like
`test_[a function]_happy_path()`
which is a test that the expected behavior occurs when good inputs are fed to the function,
or
`test_[a function]_[some description of bad inputs]()`

so for instance, this test would be called
`test_clean_canvas_happy_path()`.

Reply:
Thanks, Dave! This is very helpful. I appreciate it. 

The Canvas object takes in two inputs (catalog and nside) and it's important that they have the correct type. So it'd be a good idea to add tests for the "bad inputs" there!  I'll implement this soon.